src/node/win.ts

```diff
@@ -7,6 +8,8 @@ import { BrowserWindow } from "./browser-window";
  */
 export class Win implements IWindowProvider {
   public async createWindow(opts: IWindowConstructorOpts) {
+    const logger = pino();
+    const originalURL = new URL(opts.url);
```

Comment:
should we be using `url` from the node url module? or this browser-builtin?

Reply:
Not sure about this. Can you let me know what are the two URL classes that you're mentioning? Which one do you prefer?

Reply:
my understanding is electron uses [url](https://nodejs.org/api/url.html) under the hood - this seems to be [URL](https://developer.mozilla.org/en-US/docs/Web/API/URL) which is seemingly different. Probably best to use the one electron uses, which afaik is `import url from url` and then `url.parse()`, etc.

Reply:
actually - looks like node has updated, and these are designed to have feature parity, so won't block on it. https://nodejs.org/api/url.html#url_the_whatwg_url_api